codetransformer/transformers/patterns.py

```diff
@@ -0,0 +1,103 @@
+from sys import exc_info
+
+from dis import (
+    cmp_op,
+    dis,
+)
+from enum import (
+    IntEnum,
+    unique,
+)
+
+
+from codetransformer.core import CodeTransformer
+from codetransformer.instructions import (
+    CALL_FUNCTION,
+    CALL_FUNCTION_VAR,
+    POP_TOP,
+    ROT_TWO,
+)
+
+
+@unique
+class Comparisons(IntEnum):
```

Comment:
I know there is a place in lazy where I have a magic int for 'is' and a comment. I wonder if this should move to instruction, or maybe make this an attribute of COMPARE_OP. I'm not sure what's best.